src/py/flwr/cli/config_utils.py

```diff
@@ -145,8 +137,6 @@ def validate_fields(config: Dict[str, Any]) -> Tuple[bool, List[str], List[str]]
     else:
         if "publisher" not in config["tool"]["flwr"]["app"]:
             errors.append('Property "publisher" missing in [tool.flwr.app]')
-        if "config" in config["tool"]["flwr"]["app"]:
-            _validate_run_config(config["tool"]["flwr"]["app"]["config"], errors)
```

Comment:
We cannot remove this validation. We should instead update it to check if only allowed value types are configured.

Reply:
Good point, I'll update this